cloudferrylib/os/actions/check_sql.py

```diff
@@ -13,10 +13,23 @@
 # limitations under the License.
 
 from cloudferrylib.base.action import action
+from cloudferrylib.os.compute import nova_compute
+from cloudferrylib.os.image import glance_image
+from cloudferrylib.os.storage import cinder_database
+from cloudferrylib.utils import utils as utl
+
+SQL = "SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES LIMIT 1"
```

Comment:
Let's keep the shortest possible lifetime for data, please move this back to the inner space where we actually use this sql, so that it gets destroyed once not needed.